accelerator/core/serializer/serializer.c

```diff
@@ -694,7 +717,192 @@ status_t ta_send_transfer_res_serialize(ta_send_transfer_res_t* res, char** obj)
   return ret;
 }
 
-status_t receive_mam_message_res_serialize(char* const message, char** obj) {
+static status_t recv_mam_message_mam_v1_req_deserialize(cJSON const* const json_obj, ta_recv_mam_req_t* const req) {
+  cJSON *json_sub = NULL, *json_value = NULL;
+  status_t ret = SC_OK;
+  char *bundle_hash = NULL, *chid = NULL, *epid = NULL, *msg_id = NULL, *psk = NULL, *ntru = NULL;
+
+  if (cJSON_HasObjectItem(json_obj, "key")) {
+    json_sub = cJSON_GetObjectItem(json_obj, "key");
+    if (json_sub == NULL) {
+      ret = SC_SERIALIZER_JSON_PARSE;
+      ta_log_error("%s\n", "SC_SERIALIZER_JSON_PARSE");
+      goto done;
+    }
+  }
+
+  if (cJSON_HasObjectItem(json_sub, "psk")) {
```

Comment:
Can we combine `ntru` and `psk` into one `key` column? 
Since the two keys won't be used at the same time,and we can know which key is used by the length.
That way we can simplify the code here and elsewhere that is also related to `key`.

Reply:
Got it